msg/DetailedStatus.msg

```diff
@@ -0,0 +1,6 @@
+# The configured area number the stop occurred in.
```

Comment:
Is this message used anywhere? I didn't see anything creating a DetailedStatus object in this PR.

Reply:
This was a placeholder for a new message that should be requested in a different channel. But I will remove it as the definition is not complete.

Reply:
The initial goal was to do something like this:
![image](https://user-images.githubusercontent.com/77983813/181515202-d12de024-c8a8-4ae9-850c-0f8fa3770b80.png)